students/17th/team1/gyuboshim/westagram/user/views.py

```diff
@@ -0,0 +1,77 @@
+from django.shortcuts   import render
+from django.views       import View
+from django.http        import HttpResponse, JsonResponse
+from django.db.models   import Q
+
+from .models            import User
+
+import json
```

Comment:
```suggestion
import json

from django.shortcuts   import render
from django.views       import View
from django.http        import HttpResponse, JsonResponse
from django.db.models   import Q

from .models            import User
```
임포트 순서는 다음과 같습니다. 
1. 파이썬 모듈
2. 장고 모듈
3. 사용자화된 모듈 
각 종류별 한줄띄우기로 구분해서 임포트 정리해주세요.